### ISSUE TITLE
GURA001 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Roslyn.Asserts#155]
```cs
            [Test]
            public static void Stylet()
            {
                var code = @"
namespace RoslynSandbox
{
    public class Foo : Stylet.PropertyChangedBase
    {
        public int Bar { get; set; }
    }
}";

                RoslynAssert.Valid(Analyzer, code, metadataReferences: SpecialMetadataReferences.Stylet);
            }
```
